src/ck-perf/trace-common.C

```diff
@@ -264,11 +264,13 @@ void traceWriteSTS(FILE *stsfp,int nUserEvents) {
 #if CMK_SMP_TRACE_COMMTHREAD
```

Comment:
Could this code be responsible for the empty row that is present if you pass `--enable-tracing-commthread` to a multicore build, and if so, would this fix it?

```suggestion
#if CMK_SMP_TRACE_COMMTHREAD && !CMK_SMP_NO_COMMTHD
```

Reply:
Yeah, that should, although I'd say we should probably prevent building like that in the first place, giving a build time error when someone tries to enable comm thread tracing when there is no comm thread.